src/stories/Forms/Checkbox/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+import { GlobalInputProps } from '../../../shared/types';
+
+interface WrapperProps {
+  hasError?: boolean;
+}
+
+const Wrapper = styled.div<WrapperProps>`
+  label {
+    display: flex;
+    font-size: var(--font-size-14);
+    cursor: pointer;
+
+    // Checkbox
+    &:before {
+      display: block;
+      flex-shrink: 0;
+      width: var(--spacing-16);
+      height: var(--spacing-16);
+      margin-right: var(--spacing-8);
+      border-radius: 2px;
+      content: '';
+
+      border: ${({ hasError }) =>
+        hasError ? 'var(--border-error)' : 'var(--border-primary)'};
+    }
+
+    a {
+      color: var(--color-lavender);
+    }
+  }
+
+  input[type='checkbox'] {
+    opacity: 0;
+    position: absolute;
+    left: -99999px; // This is to hide the checkbox without affecting screen readers
+
+    // Focus ring
+    &:focus + label:before {
+      outline: 1px dotted #212121; // Fallback to non-webkit browsers
+      outline: 5px auto -webkit-focus-ring-color;
+      outline-offset: 3px;
+    }
+
+    // Checked checkbox style
```

Comment:
One minute silence for the loss of our one liner 💀

Reply:
![https://c.tenor.com/bl_3kYhSzsgAAAAM/spongebob-rip-sad-sponge-bob.gif](https://c.tenor.com/bl_3kYhSzsgAAAAM/spongebob-rip-sad-sponge-bob.gif)

Reply:
😂